colcon-test/README.md

```diff
@@ -0,0 +1,84 @@
+# colcon-test
+
+This action runs `colcon test` with labels specified in a regex format.
+Note that you need to build target packages before running this action.
+
+## Usage
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: ros:galactic
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get modified packages
+        id: get-modified-packages
+        uses: autowarefoundation/autoware-github-actions/get-modified-packages@tier4/proposal
+
+      - name: Build
+        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+        uses: autowarefoundation/autoware-github-actions/colcon-build@tier4/proposal
+        with:
+          rosdistro: galactic
+          target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
+          build-depends-repos: build_depends.repos
+
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    container: ros:galactic
+    strategy:
+      matrix:
+        label:
+          - gtest
+          - launch_test
```

Comment:
How about preparing two variables here, for example, `test-label` and `codecov-flags`.

Probably like this?

```suggestion
      include:
        - test-label: gtest
          codecov-flags: gtest

        - test-label: launch_test
          codecov-flags: launch_test
```

Reply:
I think it works, I just need to fix the readme.

Reply:
d001cbd